pkgs/by-name/em/em100/package.nix

```diff
@@ -0,0 +1,48 @@
+{
+  curl,
+  fetchgit,
+  lib,
+  libusb1,
+  pkg-config,
+  stdenv,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "em100";
+  version = "unstable-2024-11-14";
+
+  src = fetchgit {
+    url = "https://review.coreboot.org/em100";
+    rev = "a78b4ba4774f05ecd7af495604b437113596d70e";
```

Comment:
Where does this rev come from? Please document e.g. the branch this is taken from.